ChangeLog

```diff
@@ -1,5 +1,10 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x branch released xxxx-xx-xx
+
+   * Fix compilation error on MINGW32/MINGW64 with deactivated option
+     `MBEDTLS_HAVE_ASM` in config file (@merlokk).
```

Comment:
Minor: We don't add ticks around identifiers in the ChangeLog.

Reply:
done